cassiopeia/data/CassiopeiaTree.py

```diff
@@ -688,6 +688,10 @@ def get_newick(self) -> str:
         """Returns newick format of tree."""
         return utilities.to_newick(self.__network)
 
+    def get_network(self) -> nx.DiGraph:
```

Comment:
This is intended to return the networkx topology, though I could imagine a user wanting all the information as node attributes. Then again, why not just use the CassiopeiaTree?

What are your thoughts here on what this should return?

Reply:
I agree, I would just commit to returning the tree topology. My only proposal is to perhaps rename this to `get_tree_topology` to be more explicit and the user to not have any surprises.

Reply:
Hmm. I think that's a bit unclear though since `get_network` connotes what you're expecting to get out. I'm open to changing it downstream - let's talk about it at our next standup!

Reply:
I don't have strong opinions about naming. I only think we should be clear about what is being returned: can they assume they'll get branch lengths, node times, etc. out of this? I think we should only guarantee the topology, even though it might have other attributes populated.

Reply:
This is a good point. I think you're right actually - I'll change the function to `get_tree_topology`. It would be too confusing, I think, to enumerate where exactly all of these data fields live in the networkx object being returned. 

I'll leave a TODO for us saying that it would be nice to have a function that will return a "rich" topology in networkx format with all of these attributes like branch lengths & node times.